Content/SandboxieTrace.md

```diff
@@ -14,47 +10,70 @@ Note that in this case, the sandboxed program is not creating the resource itsel
+- **DebugTrace** (Log Debug Output to Trace Log) some applications emit messages to the Windows debug log by default.  Checking this option (or setting it to *y* in the ini) will redirect application debug messages to show up in the Sandboxie Trace Log.
+- **ErrorTrace** (Log all SetError's to Trace log) when a Win32 API call is executed it often doesn't return the exact error it runs into, instead it calls SetError with the error code.  The application making the call would normally call GetLastError to return the last error set.  Checking this option (or setting it to 'y' in the ini) will log all SetError calls to the Sandboxie Trace Log.  Note: Many calls log plenty errors that are expected or handled, and there are a large number of Win32 calls made by most programs.  Enabling this option will result in a lot of output.
+- **LogAPI** library to get additional trace output (see [this thread](https://forum.xanasoft.com/viewtopic.php?f=12&t=143) for more information).
+- **DisableResourceMonitor** this does not actually enable any logging but instead turns off the default trace options that are enabled any time trace logging is turned on.  This can be useful if you explicitly only want to see the output from the trace options you enable.  To enable this check the box or set it to *y* in the ini.
 
-*   **FileTrace** logs access to files, folders, and filesystem volumes;
-*   **KeyTrace** logs access to registry keys (but not values within keys);
-*   **PipeTrace** logs access to named pipes and mail slot objects which are used for inter-process communication;
-*   **IpcTrace** logs access to other objects used for inter-process communication, and also logs access attempts by one process to another process;
-*   **GuiTrace** logs window-to-window communications;
-*   **ClsidTrace** logs COM communications;
-*   **NetFwTrace** traces the actions of the firewall components (since version 0.9.0 / 5.51.0);
-*   **LogAPI** library to get additional trace output (see [this thread](https://forum.xanasoft.com/viewtopic.php?f=12&t=143) for more information).
+If you configure these options using the [Sandboxie Agent](SandboxieAgent.md) they will be configured to log all requests (both successful and failed request).  If you want to only log one result you can edit the setting in the ini.  In the ini each setting accepts a sequence of characters which specifies what to log. The wildcard character \* logs all requests;  the character *a* logs requests which were allowed; the character *d* logs requests which were denied. For the **FileTrace** and **PipeTrace** settings, the character *i* logs requests which were allowed because they access a device which is ignored by Sandboxie, such as a CD-ROM.   For **DisableResourceMonitor**, **ErrorTrace** and **DebugTrace** the only valid value is *y* for enabled.  For **LogAPI** it requires specific DLL Inject lines to enable, see the thread above (or use the UI Agent).

```

Comment:
Even if David tends to use the Sandboxie Agent definition, I think Sandboxie UI would be more appropriate to a wider audience.